pkg/kapp/diffgraph/change_graph_test.go

```diff
@@ -590,6 +591,110 @@ roleRef:
 	require.Equal(t, expectedOutput, output)
 }
 
+const appCRSvcAccntRBACYaml = `
```

Comment:
There are two APIs which are using this YAML value. If will move inside the API, it will be pasted twice. I think keeping it outside is better than pasting twice same thing.

Reply:
Hmm, should we have just one test with both upsert and deletion?